app/scripts/controllers/membership.js

```diff
@@ -27,7 +27,7 @@ angular
 
       // NOTE: these could all be moved out into a strings service.
       var messages = {
-        errorReason: _.template('Reason: "<%- httpErr %>"'),
+        errorReason: _.template('Reason: "<%= httpErr %>"'),
```

Comment:
We shouldn't keep some plain text and some HTML markup messages in the same `messages` object. It's going to get too confusing to know when the message needs to be escaped. I'd create separate messages objects for alerts / html dialog messages. Or simply inline the messages.